### PR TITLE
[#28] A page for the learners group

### PR DESCRIPTION
**Pull request by ioctaptceb**
Hey, 

PR re: #28 
Took me a moment to understand what files were going where. I did two "versions" of this. one with the two column layout, the other with the simple layout. They are both here. I am not so familiar with the best way to do things so input is much appreciated. @vsmart @lislis 

Also, the German is just filler for now. If I get comments today, I will try to update the pr today.

Thanks! sorry for the delay

### CODE REVIEW

**Comment by lislis** [comment]
I'm more in favour of the two-cloumn version. More in-line with the landing page :)

**Comment by lislis** [comment]
I took the liberty to push directly onto your branch :)

**Comment by ioctaptceb** [comment]
we can delete the .md then

**Comment by vsmart** [comment]
Hey @lithop! I checked out the page and it looks great so far 👏 

Some small comments: 
- please link to this side on the start page too, perhaps under "Resources" 
- I would put an intro sentence somewhere at the top. Something like this: 

"The learning group is a collective of women and non binary people interested in programming. We meet one evening a week to learn Clojure, facilitated by coaches (of various genders)."
Auf deutsch "Die Lerngruppe ist ein Kollektiv von Frauen und nichtbinären Menschen, die am Programmieren lernen interessiert sind. Wir treffen uns einen Abend in der Woche. Mit dabei sind Coaches (verschiedenen Genders), die uns zur Unterstützung stehen."  

After that I think we can merge this. Can always iterate more later. 👍

**Comment by Malwine** [comment]
Hi dear people, first of all: Thanks for the great work, everybody!
2 cents from my side: I suggest to have the permalink a dash "-" instead of an underscore "_". 

Moreover we started to do gender specific language on the main page with a capital I in the middle. I would adjust this. Currently you use a underscore. 
We can definitely discuss which notation to use in the future but we need to be consistent for now. I think this is quite important.

File: learners_group.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            <p>Ort: Berlin; Die genaue Adresse kann variieren, am besten ist es direkt über Slack mit uns in Verbindung zu treten <a href='mailto:info@clojurebridge-berlin.org?subject=ClojureDojo slack'> (Klick hier!) </a></p>
+          </li>
+          <li>
+            <p>Wann: Jeden Mittwoch im Monat außer am zweiten Mittwoch (dann gehen wir zum Clojure Meetup)</p>
+          </li>
+          <li>
+            <p>Was: Du kannst entweder auf eigene Faust arbeiten oder Menschen mit gemeinsamen Interessen finden und gemeinsam an einem Projekt über einen längeren Zeitraum arbeiten</p>
+          </li>
+          <li>
+            <p>Wir beginnen die Lerngruppe mit einem veganen Abendessen, bei dem jede_r etwas mitbringt (engl. Potluck), du kannst gern auch eine Kleinigkeit mitbringen und mit den anderen teilen.</p>
```
Comment (Malwine) [inline learners_group.html]: I suggest "alle etwas mitbringen" instead of "jede_r"

File: learners_group.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      </div>
+
+      <div class="col-md-6">
+        <p>
+          <b>Deutsch: </b>
+        </p>
+        <p>Du möchtest anfangen Clojure zu lernen oder damit weitermachen?</p>
+        <p>Werde Teil unserer Lerngruppe!</p>
+        <p>Anfänger_innen sind absolut willkommen, falls du am Workshop teilgenommen hast weißt du mehr als genug.<br>
+          Selbst wenn du nicht beim Workshop warst sind genug andere Lernende da die dir gern dabei helfen können loszulegen.
```
Comment (Malwine) [inline learners_group.html]: A comma is missing after "genug Lernende da" and before "die". -> "genug Lernende da, die"

**Comment by ioctaptceb** [comment]
I will take care of this either today in the evening or tomorrow!

**Comment by jellea** [comment]
This is great! Thanks @lithop. Shall we also mention the CoC on this page? And maybe add some of the previously made art to the page? :D

**Comment by vsmart** [comment]
Hi @lithop, I would like to merge this before the Installfest tomorrow. If you don't have time to incorporate the latest feedback, I think that's fine. The first version is good and we can iterate on it later. What do you think?

**Comment by lithop** [comment]
@vsmart thanks for the reminder. I forgot the last few evenings, and I don't have time tonight so I think its best to merge now, iterate later

**Comment by lithop** [comment]
I will collect this feedback in a ticket now, but you can take care of the merging

**Comment by ioctaptceb** [comment]
I did not manage to get to the art, but we can do that in the next version. Took care of wording and linking. @vsmart please feel free to merge as you see fit

**Comment by vsmart** [comment]
@lithop WHEE! Thank you! ✨ 💖

**Pull request merged**